modules/util/util.go

```diff
@@ -191,3 +192,8 @@ var titleCaser = cases.Title(language.English)
 func ToTitleCase(s string) string {
 	return titleCaser.String(s)
 }
+
+// Dedent removes common indentation of a multi-line string along with whitespace around it
+func Dedent(s string) string {
+	return strings.TrimSpace(dedent.Dedent(s))
```

Comment:
This seems like a small function perfectly eligible for copying instead of adding a new dependency. I think we should rather do that.

Reply:
Done, although I think keeping it as dependency would have been perfectly fine as well.